webhook/resources/volume/mutator.go

```diff
@@ -97,7 +97,24 @@ func (v *volumeMutator) Create(request *admission.Request, newObj runtime.Object
 	}
 
 	if string(volume.Spec.AccessMode) == "" {
-		patchOps = append(patchOps, fmt.Sprintf(`{"op": "replace", "path": "/spec/accessMode", "value": "%s"}`, string(longhorn.AccessModeReadWriteOnce)))
+		accessModeFromBackup := longhorn.AccessModeReadWriteOnce
+		if volume.Spec.FromBackup != "" {
+			bName, _, _, err := backupstore.DecodeBackupURL(volume.Spec.FromBackup)
+			if err != nil {
+				return nil, werror.NewInternalError(fmt.Sprintf("cannot get backup and volume name from backup URL %v: %v", volume.Spec.FromBackup, err))
+			}
+			bs, err := v.ds.GetBackupRO(bName)
+			if err != nil {
+				return nil, werror.NewInternalError(fmt.Sprintf("failed to list backups for volumes: %v", bName))
+			}
+
+			if bs != nil {
```

Comment:
`bs != nil` should not necessary, right?

Reply:
Yes, I think you are right. 

Since I would check if the tab is exited, I don't think checking bs is necessary.

I will remove the redundant check. Thank you.